src/core/ledger/distributionSummary/distributionSummary.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+
+import {Ledger} from "../ledger";
+import {type Distribution} from "../distribution";
+import * as G from "../grain";
+import sortBy from "../../../util/sortBy";
+import {type IdentityId} from "../../identity";
+import * as NullUtil from "../../../util/null";
+import {type CurrencyDetails} from "../../../api/currencyConfig";
+
+export function distributionMarkdownSummary(
+  distribution: Distribution,
+  ledger: Ledger,
+  currencyDetails: CurrencyDetails
+) {
+  const {name: currencyName, suffix: currencySuffix} = currencyDetails;
+  const distributionBalances: DistributionBalances = getDistributionBalances(
+    distribution
+  );
+  const totalDistributed: G.Grain = getTotalDistributed(distributionBalances);
+
+  const ids = Array.from(distributionBalances.keys());
+  const countIdentities = ids.length;
+
+  console.log(`## ${currencyName.toUpperCase()} Distribution`);
```

Comment:
i dont think this line adds much value at present. however, might be nice to include the cred timestamp of the distribution (in human readable form). that's helpful esp. if there are multiple distributions.

Reply:
Ack, left this custom currency line in, but added a cred timestamp as you suggested.